src/editor/lexercpp.h

```diff
@@ -13,9 +13,9 @@
 class LexerCPP: public iLexer
 {
 private:
-    QVector<QVector<Token>> mTokens;
     QVector<Token> mTokensOnCurrentLine;
     State mState;
+    bool isCommentBlock;
```

Comment:
1)do you use this variable?
2)mIsCommentBlock.
3) Why is it variable?
4) Why do you named variable as a method?